src/Tracing/Span.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tracing;
+
+use Exception;
+
+class Span
+{
+    private $name;
+    private $spanContext;
+    private $parentSpanContext;
+
+    private $start;
+    private $end;
+    private $status;
+
+    private $attributes = [];
+    private $events = [];
+
+    public function __construct(string $name, SpanContext $spanContext, SpanContext $parentSpanContext = null)
```

Comment:
Nit: the technically correct type is: `?SpanContext`. The `?` is allowed to be omitted for now, but I would not be surprised at all if in the future it was required, or if linters/PSR type things started requiring it.

Reply:
+1 for this; following the spec here would be smart for future compatibility.